templates/configurations/slider.interface.hbs

```diff
@@ -0,0 +1,4 @@
+<div v-show="defaultValueSource === 'default'" class="form-group">
+  <label>Default value</label>
+  <input class="form-control" v-model.trim="value" placeholder="Default value"></input>
```

Comment:
```suggestion
  <input class="form-control" v-model.trim="value" placeholder="Default value">
```

Reply:
did the changes.

Reply:
Doesn't look like this was resolved. Where did you make the changes?

Reply:
Hi @tonytlwu, while fixing PR other feedback, accidentally it got reverted.